### DISCUSSION TITLE
Can I change the image switching speed of fancybox

### DISCUSSION THREAD

**Discussion by colormango** [fancyapps/ui#251] [Q&A]
I tested and found that in iPhone, Fancybox's image switching speed is a bit slow, making people feel like losing frames, how can I change the Fancybox default switching speed?

**Comment by fancyapps** [comment]
Hi,

You can tweak animation using `Carousel.friction` option, like this:

```js
Fancybox.bind('[data-fancybox="gallery"]', {
  Carousel :{
    friction: 0.5
  }
});
```

You can experiment with the value to get desired animation duration, something around 0.8 would probably be best.
